Core/src/main/java/com/plotsquared/core/database/SQLManager.java

```diff
@@ -154,13 +155,20 @@ public SQLManager(
         this.worldConfiguration = worldConfiguration;
         this.database = database;
         this.connection = database.openConnection();
+        this.supportsGetGeneratedKeys = this.connection.getMetaData().supportsGetGeneratedKeys();
         this.mySQL = database instanceof MySQL;
         this.globalTasks = new ConcurrentLinkedQueue<>();
         this.notifyTasks = new ConcurrentLinkedQueue<>();
         this.plotTasks = new ConcurrentHashMap<>();
         this.playerTasks = new ConcurrentHashMap<>();
         this.clusterTasks = new ConcurrentHashMap<>();
         this.prefix = prefix;
+
+        if (mySQL && !supportsGetGeneratedKeys) {
+            throw new SQLException("Database Driver for MySQL does not support Statement#getGeneratedKeys - which breaks " +
+                    "PlotSquared functionality");
```

Comment:
Is it beneficial to dump the driver version too? In case there are more breaking changes in the future, between paper and spigot.

Reply:
Could be a great addition, especially for debugging purposes